cli/tests/integrations/test_marathon.py

```diff
@@ -829,7 +829,7 @@ def _zero_instance_app():
 def _zero_instance_app_through_http():
     class JSONRequestHandler (BaseHTTPRequestHandler):
 
-        def do_GET(self):
+        def do_GET(self):  # noqa: N802
```

Comment:
Yes, and it ignores only the pep8 naming check. The `_GET` and friends of the BaseHTTPRequestHandler sadly do not follow pep8, so have to silence it.